### DISCUSSION TITLE
Change starting point

### DISCUSSION THREAD

**Discussion by manuelmeister** [Yqnn/svg-path-editor#76] [Ideas]
It would be cool to rearrange the command list so that the starting point of a shape could be something more sensible. Like instead of between to "connecting" arcs, it could be at an edge. With this, i could reduce one or even two commands (as in the example picture i could replace the two arcs with one and the two line commands with a single line and a Z.

From:
![Screenshot of a shape like a piece of cake](https://user-images.githubusercontent.com/3001985/163956854-2d495e6d-11b7-4b77-974a-1dbe5800c662.png)

To:
![Same shape but with fewer commands](https://user-images.githubusercontent.com/3001985/163958829-be69d440-457c-4f40-aa93-c25506d34a8d.png)

**Comment by Yqnn** [comment]
Same as issue #10
